Tests/BootCamp.Chapter.Tests/CredentialsTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using BootCamp.Chapter.Tests.Input.ClassData;
+using FluentAssertions;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class CredentialsTests
+    {
+        [Theory]
+        [InlineData("", "")]
+        [InlineData(null, "")]
+        [InlineData("", null)]
+        [InlineData(null, null)]
+        public void New_Credentials_Given_Username_NullOrEmpty_Throws_ArgumentException(string username, string password)
+        {
+            Action action = () => new Credentials(username, password);
+
+            action.Should().Throw<ArgumentException>();
+        }
+
+        [Fact]
+        public void Equals_On_Different_Type_Returns_False()
+        {
+            int objectOfDifferentType = default;
```

Comment:
Codacy found an issue: [Add the 'const' modifier to 'objectOfDifferentType'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5768165)